webapp-django/crashstats/crashstats/models.py

```diff
@@ -629,6 +629,10 @@ class TelemetryCrash(SocorroMiddleware):
     required_params = ("crash_id",)
     aliases = {"crash_id": "uuid"}
 
+    post = None
+    put = None
+    delete = None
```

Comment:
We never use PUT or DELETE, so I don't understand why there's any support for them. I almost removed them, but then I decided not to. Future me will probably regret that. I may rethink this choice tomorrow.